src/detect/ScanI2CTwoWire.cpp

```diff
@@ -162,13 +162,13 @@ void ScanI2CTwoWire::scanPort(I2CPort port, uint8_t *address, uint8_t asize)
     Melopero_RV3028 rtc;
 #endif
 
-#ifdef I2C_SDA1
+#if defined(I2C_SDA1) || (defined(NRF52840_XXAA) && (WIRE_INTERFACES_COUNT == 2))
     if (port == I2CPort::WIRE1) {
```

Comment:
feels like the (defined(NRF52840_XXAA) && (WIRE_INTERFACES_COUNT == 2)) can be combined into a single macro, or even just define I2C_SDA1 - as it is now there's a lot of duplicate code.

Reply:
I did try out the idea of defining `I2C_SDA1`, but it messed with the preprocessor logic for the `Wire.begin`.
I do agree though, it's messy. I'll go back and see what options I can come up with to tidy it.

Reply:
I'm thinking either:
 ```cpp
#if defined(PIN_SDA1) || defined(PIN_WIRE1_SDA)
```
or
```cpp
#if WIRE_INTERFACES_COUNT == 2
```
and setting `WIRE_INTERFACES_COUNT` once in configuration.h

What are your thoughts on these as options?